pkg/urbit/worker/main.c

```diff
@@ -122,6 +124,22 @@ void _worker_space(FILE* fil_u,  c3_w n)
     (fprintf(fil_u," "));
 }
 
+/* _worker_check_zuse(): check arvo compatibility.
+*/
+void
+_worker_check_zuse(u3_noun ver)
+{
+  if ( ver > _CVX_ZUSE ) {
+    u3l_log("work: zuse incompatible (need vere < v" URBIT_VERSION ")\r\n");
```

Comment:
I think we *should* print the version numbers here. Have/need or something like that.

Reply:
I should add, these printfs should use `stderr` over `u3l_log()`; the latter goes over the pipe, and we'll exit immediately, so they might not get there.